airflow/operators/python.py

```diff
@@ -599,6 +611,25 @@ def _iter_serializable_context_keys(self):
             yield from self.PENDULUM_SERIALIZABLE_CONTEXT_KEYS
 
 
+class BranchPythonVirtualenvOperator(PythonVirtualenvOperator, SkipMixin):
```

Comment:
Should we separate it out to a separate PR @jens-scheffler-bosch ? That seems rather unrelated and it would be nice to  have it as a separate changelog entry.

Reply:
Besides - there are missing tests for this one it seems

Reply:
Separated-out the BranchPythonVirtualenvOperator to https://github.com/boschglobal/airflow/pull/3 - and you are right - in deep I need to think about how I could add a pytest. Not sure how this can be made test-able.
Also a decorated partner feature would be good if I think about it.